src/app/settings/settings.module.ts

```diff
@@ -0,0 +1,33 @@
+import { CommonModule } from "@angular/common";
+import { NgModule } from "@angular/core";
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { IonicModule } from "@ionic/angular";
+import { TranslateModule } from "@ngx-translate/core";
+import { NgxsModule } from "@ngxs/store";
+
+import { CustomNetworkCreateModal } from "@/app/modals/custom-network-create/custom-network-create";
+import { CustomNetworkCreateModalModule } from "@/app/modals/custom-network-create/custom-network-create.module";
+import { ViewerLogModalModule } from "@/components/viewer-log/viewer-log.modal.module";
+
+import { SettingsRoutingModule } from "./settings-routing.module";
+import { SettingsPage } from "./settings.component";
+import { SettingsService } from "./shared/settings.service";
+import { SettingsState } from "./shared/settings.state";
+
+@NgModule({
+	declarations: [SettingsPage],
+	providers: [SettingsService],
```

Comment:
will be injected by Angular automatically due to lazy loading.

```suggestion
	providers: [],
```